### DISCUSSION TITLE
Error after configuring rust language server

### DISCUSSION THREAD

**Discussion by Yakumo-Yukari** [helix-editor/helix#10298] [Troubleshooting]
`configure`

```
[[language]]
name = "rust"
scope = "source.rust"
injection-regex = "rust"
file-types = ["rs"]
roots = ["Cargo.toml", "Cargo.lock"]
auto-format = true
comment-tokens = ["//", "///", "//!"]
block-comment-tokens = [
    { start = "/*", end = "*/" },
    { start = "/**", end = "*/" },
    { start = "/*!", end = "*/" },
]
language-servers = ["rust-analyzer"]
indent = { tab-width = 4, unit = "    " }
persistent-diagnostic-sources = ["rustc", "clippy"]
```

`error`

Bad language config: unknown field `block-comment-tokens`, expected one of `name`, `language-id`, `scope`, `file-types`, `shebangs`, `roots`, `comment-token`, `text-width`, `soft-wrap`, `auto-format`, `formatter`, `diagnostic-severity`, `grammar`, `injection-regex`, `language-servers`, `indent`, `debugger`, `auto-pairs`, `rulers`, `workspace-lsp-roots`
in `language`

Press <ENTER> to continue with default language config

**Comment by kirawi** [comment]
It looks like you have 23.10 instead of 24.03. If you're copying the languages.toml from master, you don't need to since the config is merged. You're also not configuring the LSP correctly. It should just be

```toml
[language-server.rust-analyzer.config]
persistent-diagnostic-sources = ["rustc", "clippy"]
```
